content/docs/2.15/scalers/datadog.md

```diff
@@ -141,7 +141,10 @@ spec:
       key: token
     - parameter: datadogNamespace
       name: datadog-config
-      key: namespace
+      key: datadogNamespace
+    - parameter: datadogMetricsService
+      name: datadog-config
+      key: datadogMetricsService
```

Comment:
Please don't add this, it is already specifeid below

Reply:
I've been done.

Reply:
@zroubalik something missing in this PR?